tutorials/vue/resize.md

```diff
@@ -0,0 +1,130 @@
+To create a resize event, you have a couple options.
+
+## Add an Event Listener
+
+This is the simpler approach of the two. You must utilize the
+`created()` and `destroyed()` lifecycle hooks to invoke the `addEventListener()`
```

Comment:
"You should register an event listener for the window `'resize'` event using `addEventListener()` when Vue mounts the component. You should also clean up the event listener when the component is unmounted."

Also, I'd recommend using `mounted()` and `unmounted()` over `created()` and `destroyed()`.